com.pot_app.pot.yml

```diff
@@ -0,0 +1,177 @@
+id: com.pot_app.pot
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: pot
+rename-icon: pot
+rename-desktop-file: pot.desktop
+
+finish-args:
+  - --socket=wayland
```

Comment:
Does it actually work on wayland? You add xrandr and xdotool

Reply:
It does work on Wayland, but it's still needed to run on X11